Cargo.toml

```diff
@@ -43,6 +43,8 @@ uuid = { version = "^1", default-features = false, optional = true }
 ouroboros = { version = "0.15", default-features = false }
 url = { version = "^2.2", default-features = false }
 thiserror = { version = "^1", default-features = false }
+tokio = { version = "^1.6", default-features = false, features = ["sync"] }
+async-channel = { version = "^1.7", default-features = false }
```

Comment:
This is not a broadcast channel, as each message will only be received by one consumer.